orcid-persistence/src/main/java/org/orcid/persistence/dao/impl/WorkDaoImpl.java

```diff
@@ -400,7 +405,6 @@ public List<Object[]> getWorksByOrcid(String orcid) {
                 .addScalar("assertion_origin_client_source_id", StringType.INSTANCE)
                 .addScalar("date_created", TimestampType.INSTANCE)
                 .addScalar("last_modified", TimestampType.INSTANCE)
-                .addScalar("contributors", StringType.INSTANCE)
```

Comment:
@DanielPalafox are you sure we dont need the scalar anymore?

Reply:
Yes we need it you are rigth!. I already fix it!